### PR TITLE
Update paypal-transaction.md

### PR DESCRIPTION
**Pull request by Jgerardopine**
Correction on page_size

## What
*Describe what the change is solving*
*It helps to add screenshots if it affects the frontend.*

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- Connector version is set to `0.0.1`
    - `Dockerfile` has version `0.0.1`
- Documentation updated
    - Connector's `README.md`
    - Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - `docs/integrations/<source or destination>/<name>.md` including changelog with an entry for the initial version. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - `docs/integrations/README.md`

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

<details><summary><strong>Connector Generator</strong></summary>

- Issue acceptance criteria met
- PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/resources/pull-requests-handbook)
- If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:generateScaffolds` then checking in your changes
- Documentation which references the generator is updated as needed

</details>

<details><summary><strong>Updating the Python CDK</strong></summary>

### Airbyter

Before merging:
- Pull Request description explains what problem it is solving
- Code change is unit tested
- Build and my-py check pass
- Smoke test the change on at least one affected connector
   - On Github: Run [this workflow](https://github.com/airbytehq/airbyte/actions/workflows/connectors_tests.yml), passing `--use-local-cdk --name=source-<connector>` as options
   - Locally: `airbyte-ci connectors --use-local-cdk --name=source-<connector> test`
- PR is reviewed and approved
      
After merging:
- [Publish the CDK](https://github.com/airbytehq/airbyte/actions/workflows/publish-cdk-command-manually.yml)
   - The CDK does not follow proper semantic versioning. Choose minor if this the change has significant user impact or is a breaking change. Choose patch otherwise.
   - Write a thoughtful changelog message so we know what was updated.
- Merge the platform PR that was auto-created for updating the Connector Builder's CDK version
   - This step is optional if the change does not affect the connector builder or declarative connectors.

</details>

### CODE REVIEW

File: docs/integrations/sources/paypal-transaction.md [hunk 1]
```diff
@@ -260,7 +260,7 @@ ___
 * **Number of days per request:**  The maximum supported date range is 31 days.
 * **Historical Data:**  You can't retrieve more than 3yrs of data for the `transactions` stream. For `dispute_start_date` you can only retrieve 180 days of data (see specifications per stream) 
 * `records_per_request`: The maximum number of records in a single request are 10K (API Server restriction)
-* `page_size`: The maximum page size is 500. This has been configured by default.
+* `page_size`: The maximum page size differs in every stream. Thr Max allowed pages has been configured by default.
```
Comment (natikgadzhi) [inline docs/integrations/sources/paypal-transaction.md:263]:
```suggestion
* `page_size`: The maximum page size differs in every stream. The Max allowed pages has been configured by default.
```

File: docs/integrations/sources/paypal-transaction.md [hunk 2]
```diff
@@ -260,7 +260,7 @@ ___
...
 * `requests_per_minute` = The maximum limit is 50 requests per minute from IP address to all endpoint (API Server restriction).
```
Comment (natikgadzhi) [inline docs/integrations/sources/paypal-transaction.md:264]:
```suggestion
* `requests_per_minute`: The maximum limit is 50 requests per minute from IP address to all endpoint (API Server restriction).
```

File: docs/integrations/sources/paypal-transaction.md (hunk 1 above)
Comment (natikgadzhi) [reply docs/integrations/sources/paypal-transaction.md:263]: I don't quite understand what "the max allowed pages" is in this context, and where exactly it's configured. Can you point me to where I can learn more? Did you mean maximum page size?

Comment (Jgerardopine) [reply docs/integrations/sources/paypal-transaction.md:263]: The Maximum page size (items) per page for every stream is different. So, one stream has a page size of 20, another of 100m and so on. That is why I clarify it here, but every section has the current max configured page_size. You can check each stream section in the already publish documentation (or in this code) in the tables, or directly in paypal API documentation.

Comment (natikgadzhi) [reply docs/integrations/sources/paypal-transaction.md]: Let me clarify my question.

> The maximum page size differs in every stream. The Max allowed pages has been configured by default.

There are two different things in this paragraph: 

1. **Maximum allowed page size: you're saying it differs per stream.** My asks:
   - let's clarify that the specific `page_size` values are described below for each stream.
   - In some streams, for example [Transactions](https://github.com/airbytehq/airbyte/blob/8086648bd067e1e2118a4f6d411f01db104d4dec/docs/integrations/sources/paypal-transaction.md#transactions-stream), the `page_size` is described as `Max 500 (F)`. Can we separate configurable parameters and constants? If there's a constant value, why include "Max" with it?
2. **The Max allowed pages has been configured by default.** I think what you mean is that you've already pre-configured maximum allowed `page_size` for each stream. Not _maximum allowed number of pages_, but _maximum allowed number of items per page_. I didn't understand the sentence initially. Would you be open to reword it to "Each stream's configuration is already set to request the highest `per_page` available in the API"?

Comment (Jgerardopine) [reply docs/integrations/sources/paypal-transaction.md]: 1. I don't really understand the ask. The table contains MAX and below the description when you find an F (F: Fixed Value. This means it is not configurable.)

2. You are correct, I made a syntax mistake, it should have been "the maximum allowed number of items per page."

File: docs/integrations/sources/paypal-transaction.md [hunk 3]
```diff
@@ -260,8 +260,8 @@ ___
 * **Number of days per request:**  The maximum supported date range is 31 days.
 * **Historical Data:**  You can't retrieve more than 3yrs of data for the `transactions` stream. For `dispute_start_date` you can only retrieve 180 days of data (see specifications per stream) 
 * `records_per_request`: The maximum number of records in a single request are 10K (API Server restriction)
-* `page_size`: The maximum page size is 500. This has been configured by default.
-* `requests_per_minute` = The maximum limit is 50 requests per minute from IP address to all endpoint (API Server restriction).
+* `page_size`: The maximum page size differs in every stream. The Max allowed pages has been configured by default.
```
Comment (natikgadzhi) [inline docs/integrations/sources/paypal-transaction.md:263]:
```suggestion
* `page_size`: The number of records per page is differs per stream. `source-paypal-transaction` sets maximum allowed page size for each stream by default.
```

**Review by natikgadzhi — APPROVED** [review]
